src/pages/api/graphql.ts

```diff
@@ -58,7 +58,9 @@ const apolloServer = new ApolloServer({
   dataSources,
   typeDefs,
   resolvers,
-  tracing: process.env.NODE_ENV !== "production",
+  // TODO: do we need this? Apollo server v3 removed this but can be used this way:
+  // plugins: [require('apollo-tracing').plugin()]
+  // tracing: process.env.NODE_ENV !== "production",
```

Comment:
It has been used to trace API calls that have a waterfall structure and may take some time. Doesn't need to be enable always as far as I am considered.